app/assets/javascripts/discourse/widgets/post-menu.js.es6

```diff
@@ -52,6 +52,38 @@ export function buildButton(name, widget) {
   }
 }
 
+registerButton("read-count", attrs => {
+  if (attrs.showReadIndicator) {
+    const count = attrs.readCount;
+    if (count > 0) {
+      return {
+        action: "toggleWhoRead",
+        title: "post.controls.read_indicator",
+        className: "button-count read-indicator",
+        contents: count,
+        icon: "",
```

Comment:
I think if you don't want an icon you should be able to leave this attribute out? If that doesn't work we should adjust the button code to handle `undefined` instead of an empty string